data/db/mil-tamron.xml

```diff
@@ -1,6 +1,70 @@
 <!DOCTYPE lensdatabase SYSTEM "lensfun-database.dtd">
 <lensdatabase version="2">
 
+    <lens>
+        <maker>Tamron</maker>
+	<!-- Tamron 17-70mm F/2.8 Di III-A VC RXD -->
+        <model>E 17-70mm F2.8 B070</model>
+        <mount>Sony E</mount>
+        <cropfactor>1.534</cropfactor>
+        <aspect-ratio>4:3</aspect-ratio>
+        <calibration>
+	    <!-- Taken with Sony A6000 -->
+            <distortion model="ptlens" focal="17" a="0.02" b="-0.05815" c="0.01666"/>
+            <distortion model="ptlens" focal="24" a="0.01386" b="-0.02779" c="0.01845"/>
+            <distortion model="ptlens" focal="31" a="0.01145" b="-0.01921" c="0.02189"/>
+            <distortion model="ptlens" focal="43" a="0.00663" b="-0.00044" c="0.00186"/>
+            <distortion model="ptlens" focal="59" a="0.00423" b="-0.00244" c="0.02023"/>
+            <distortion model="ptlens" focal="70" a="0.0077" b="-0.00761" c="0.01253"/>
+            <tca model="poly3" focal="17" vr="1.0000980" vb="1.0002878"/>
+            <tca model="poly3" focal="24" vr="1.0001017" vb="1.0001101"/>
+            <tca model="poly3" focal="31" vr="1.0000655" vb="1.0000457"/>
+            <tca model="poly3" focal="43" vr="0.9999838" vb="1.0000128"/>
+            <tca model="poly3" focal="59" vr="0.9999119" vb="0.9999583"/>
+            <tca model="poly3" focal="70" vr="0.9998624" vb="0.9999315"/>
+            <vignetting model="pa" focal="17" aperture="2.8" distance="10" k1="-3.0301" k2="5.9082" k3="-3.6270"/>
+            <vignetting model="pa" focal="17" aperture="2.8" distance="1000" k1="-3.0301" k2="5.9082" k3="-3.6270"/>
+            <vignetting model="pa" focal="17" aperture="3.2" distance="10" k1="-1.7129" k2="3.4052" k3="-2.3401"/>
+            <vignetting model="pa" focal="17" aperture="3.2" distance="1000" k1="-1.7129" k2="3.4052" k3="-2.3401"/>
+            <vignetting model="pa" focal="17" aperture="3.5" distance="10" k1="-3.6480" k2="7.3186" k3="-4.3472"/>
+            <vignetting model="pa" focal="17" aperture="3.5" distance="1000" k1="-3.6480" k2="7.3186" k3="-4.3472"/>
+            <vignetting model="pa" focal="17" aperture="4" distance="10" k1="-1.9923" k2="4.1870" k3="-2.7786"/>
+            <vignetting model="pa" focal="17" aperture="4" distance="1000" k1="-1.9923" k2="4.1870" k3="-2.7786"/>
+            <vignetting model="pa" focal="17" aperture="4.5" distance="10" k1="-1.9351" k2="4.0478" k3="-2.7327"/>
+            <vignetting model="pa" focal="17" aperture="4.5" distance="1000" k1="-1.9351" k2="4.0478" k3="-2.7327"/>
```

Comment:
Sorry, I didn't notice this before, but these aperture values look wrong. The tutorial says that we should use the minimum aperture, then in 1 stop increments, then the minimum aperture. I'm not sure how important it is to get these exactly, but I'm worried that this isn't going to give good corrections at, say f/16. The recommended values would be 2.8, 4, 5.6, 8, 22 (or whatever the lens's minimum is).

Is it possible for you to update the calibration with those aperture values? (I know that's a pain, sorry) Alternatively, perhaps @bronger can comment on whether this is important.

Reply:
It was a good observation from you @gnl21 , I missed that when going through the tutorial, bummer! I'll get new pictures, i think that these f-stops that you mentioned are needed in order to get the characteristics of the lens.

Perhaps this a little bit of topic, but I want to build darktable against the latest version of lensfun, and what I've read is that the ABI has changed in lensfun. I'm just curious, how much work is needed to get it working?
Perhaps I can help out, I'm not that familiar with the lensfun nor darktable code base yet though.